ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse.yaml

```diff
@@ -35,8 +35,8 @@ tests:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
-    workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private
-- as: installer-rehearse-aws-disconnected-private-usgov
+    workflow: cucushift-installer-rehearse-aws-ipi-route53-private-hosted-zone
+- as: installer-rehearse-aws-private-usgov
```

Comment:
not sure why you changed `usgov`profiles, since this is no longer used, you can remove it. from line 39-45 https://github.com/openshift/release/blob/78a81a20b3e3b4085b3fc1c2a52944b5b23306e3/ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse.yaml#L39-L45

Reply:
Done